core/src/test/scala/unit/kafka/security/authorizer/AuthorizerTest.scala

```diff
@@ -392,161 +409,17 @@ class AuthorizerTest extends QuorumTestHarness with BaseAuthorizerTest {
     //test remove all acls for resource
     removeAcls(authorizer1, Set.empty, resource)
     TestUtils.waitAndVerifyAcls(Set.empty[AccessControlEntry], authorizer1, resource)
-    if (quorum.equals(ZK)) {
-      assertFalse(zkClient.resourceExists(resource))
-    }
 
     //test removing last acl also deletes ZooKeeper path
     acls = changeAclAndVerify(Set.empty, Set(acl1), Set.empty)
     changeAclAndVerify(acls, Set.empty, acls)
-    if (quorum.equals(ZK)) {
```

Comment:
Can we update the comment above that still mentions ZooKeeper?

Reply:
Yes, removed it.